pages/badges/badge-single.jsx

```diff
@@ -10,7 +10,11 @@ var React = require('react'),
     Divider = require('../../components/Divider.jsx'),
     BadgesAPI = require('../../lib/badges-api'),
     TeachAPI = require('../../lib/teach-api'),
-    Modal = require('../../components/modal.jsx');
+    Modal = require('../../components/modal.jsx'),
+
+    config = require('../../config/config');
+
+var ACHIEVED_BADGE_BASE_URL = config.CREDLY_BASE_URL.replace(/\/?$/, '/credit/');
```

Comment:
I kind of feel like we should not be doing this here - we should ensure the conf var has the trailing `/`, or does not have a trailing `/`, in the config object, when we read in the var from the environment for use in the app.